### PR TITLE
VIH-8362 Added force play directive 

### PR DESCRIPTION
**Pull request by robertbennett1998**
### JIRA link ###
[VIH-8362](https://tools.hmcts.net/jira/browse/VIH-8362)
[VIH-8382](https://tools.hmcts.net/jira/browse/VIH-8382)

### Change description ###
- Added a directive that will try to force play videos fixing some issues where a video is loaded when the participant hasn't interacted with the page.

**Does this PR introduce a breaking change?**
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    ngOnInit(): void {
+        this.configureVideoElement();
+        this.addEventListeners();
+    }
+
+    ngAfterViewInit(): void {
+        this.videoElement.play();
+
+        setTimeout(() => {
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts:30]: so we just attempt to play the video after 5 seconds once more? What happens if it fails to play it the second time?

**Review by ZJianWang — APPROVED** [review]
LGTM

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.html [hunk 2]
```diff
@@ -10,9 +10,11 @@ <h2 class="govuk-heading-l govuk-!-margin-top-6 govuk-!-margin-bottom-6 wr-h">
         <!-- Left hand side -->
         <div class="govuk-grid-column-one-half">
           <div class="govuk-grid-column-full">
-            <video id="preferredCameraStream" [srcObject]="selectedCameraStream" [muted]="true" autoplay playsinline width="100%" height="auto" [ngClass]="{'selfie-view': connectWithCameraOn}">
-              {{ 'select-media-devices.browser-support' | translate }}
-            </video>
+              <div style="padding: 10px; background-color: black;">
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.html:13]: Move styles to css.

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts (hunk 1 above)
Comment (robertbennett1998) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts:30]: There isn't really much else we can do. This technically should not be used it is there as a back up. As the touch start /mouse down should be the one that matters.

**Review by omio-ma — APPROVED** [review]

**Review by FarronF — APPROVED** [review]

**Pull request merged**